CDCgov/cdc-open-viz
# Feature/dev 9166

## [Replace With Ticket Number]



## Testing Steps
Added a new checkbox "Display Zero Data label" to General panel which will display a thin bar on Vertical and Horizontal bars (regular non stacked).
To test it we need to make sure we have zero value data example : 
 {
      "Location": "Work",
      "Group A": "0", // example of data as zero
      "Group B": "100",
      "Group C": "80"
    },
Go to bar chart (regular) and we should see a small thin line(bar) for zero value. Toggle checkbox "Display Zero Data label" to remove or add bar.The checkbox is checked by default.





## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

## Screenshots (if applicable)



## Additional Notes
